ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -133,17 +134,38 @@ const triggerSet: OopsyTriggerSet<Data> = {
         };
       },
     },
+    {
+      id: 'General Raise Lost Effect Tracker',
+      type: 'LosesEffect',
+      netRegex: NetRegexes.losesEffect({ effectId: '94' }),
+      run: (data, matches) => {
+        ((data.raiseLostTracker ??= {})[matches.targetId] ??= []).push(matches.timestamp);
```

Comment:
Can this just be the last timestamp, rather than an array? I suspect if two people raised at the precise same time and both events were delivered and processed in the same network packet, you'd still get lose/gain/lose/gain ordering.

Reply:
oh true, since i'm no longer using `delaySeconds` i don't need it to be an array anymore